sql/plan/insert.go

```diff
@@ -102,10 +126,17 @@ func (p *InsertInto) Execute(ctx *sql.Context) (int, error) {
 			return i, err
 		}
 
-		if err := insertable.Insert(ctx, row); err != nil {
+		rowDidNotExist, err := insertable.Insert(ctx, row)
```

Comment:
As discussed above, you should handle the ignore keyword by just ignoring any error here (and not closing the iterator). Don't try to get into the semantics of duplicate keys yet.